edalize/verilator.py

```diff
@@ -207,13 +206,9 @@ def build_main(self):
         if not "mode" in self.tool_options:
             self.tool_options["mode"] = "cc"
 
-        # Do parallel builds with <number of cpus>
-        make_job_count = multiprocessing.cpu_count()
-        args = ["-j", str(make_job_count)]
-
         if self.tool_options["mode"] == "lint-only":
             args.append("V" + self.toplevel + ".mk")
```

Comment:
This line is now broken - `args` is undefined (and unused)

Reply:
Blimey, helps if you add the file....